ui/apps/platform/src/Components/ReportJob/MyLastJobStatus.tsx

```diff
@@ -4,44 +4,44 @@ import { Spinner } from '@patternfly/react-core';
 import { ReportSnapshot } from 'services/ReportsService.types';
 import ReportJobStatus from 'Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobStatus';
 import { onDownloadReport } from 'Components/ReportJob/utils';
+import { ComplianceReportSnapshot } from 'services/ComplianceScanConfigurationService';
 
-type MyLastReportJobStatusProps = {
-    reportSnapshot: ReportSnapshot | null | undefined;
-    isLoadingReportSnapshots: boolean;
+type MyLastJobStatusProps = {
+    snapshot: ReportSnapshot | ComplianceReportSnapshot | null | undefined;
```

Comment:
+1 to Mark's comment about a common abstraction for these components, instead of the need to import from both Container-specific "Vulnerabilities" and "ComplianceScanConfService" into this Component level file. A few possibilities come to mind:

1. `Pick` from each type only the subset of properties that are needed. Doesn't really help with the Container <- Components file dependency though.
2. Create a common type here, and import that as a base type for `ReportSnapshot` and `ComplianceReportSnapshot`. This inverts the dependency to Components <- Container, which is debatable if it is better or not.
4. Create a component-specific common type here, and don't import it elsewhere. This disconnects the Components and Container dependency. A pro of this is that we aren't introducing cross-container dependencies, and that TS should detect issues should the types deviate in the future. The con is that it makes the implicit dependency a bit more difficult to track, via "Ctrl+Click", etc.
5. Create a common type somewhere else globally (`/types/`, `/services/`?). This should mitigate the cons of the approach above ^ but introduces a new abstraction that may really only be used for this component.

Reply:
Thanks for the suggestions. We can continue the conversation under Mark's comment. I think they're all related to that too